src/coredump/coredump-ratelimit.c

```diff
@@ -0,0 +1,170 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/statvfs.h>
+#include "dirent-util.h"
+#include "fd-util.h"
+#include "fs-util.h"
+#include "string-util.h"
+#include "time-util.h"
+#include "user-util.h"
+#include "util.h"
+#include "coredump-ratelimit.h"
+#include "sd-id128.h"
+
+#define MAX_COMM_LENGTH 16
```

Comment:
TASK_COMM_LEN is defined in missing_sched.h already

Reply:
Addressed the comment. Please check once.